backend/bottari/src/test/java/com/bottari/controller/BottariTemplateControllerTest.java

```diff
@@ -35,10 +35,32 @@ class BottariTemplateControllerTest {
     @Autowired
     private ObjectMapper objectMapper;
 
-    @DisplayName("모든 보따리 템플릿을 조회한다.")
+    @DisplayName("특정 보따리 템플릿을 조회한다.")
     @Test
-    void readAll() throws Exception {
+    void read() throws Exception {
         // given
+        final ReadBottariTemplateResponse response = new ReadBottariTemplateResponse(
+                1L,
+                "title_1",
+                List.of(
+                        new BottariTemplateItemResponse(1L, "item_1"),
+                        new BottariTemplateItemResponse(2L, "item_2")
+                ),
+                "author_1");
```

Comment:
```suggestion
        final ReadBottariTemplateResponse response = new ReadBottariTemplateResponse(
                1L,
                "title_1",
                List.of(
                        new BottariTemplateItemResponse(1L, "item_1"),
                        new BottariTemplateItemResponse(2L, "item_2")
                ),
                "author_1"
        );
```

readAll 테스트 코드처럼 띄우면 더 좋을 것 같아요.